packages/ui/src/components/DataStory/icons/addNodeIcon.tsx

```diff
@@ -1,5 +1,17 @@
-export function AddNodeIcon() {
-  return (<svg xmlns="http://www.w3.org/2000/svg" style={{ fill: 'none' }} viewBox="4 4 20 20" strokeWidth={2.5} stroke="currentColor" className="w-6 h-6 text-gray-700">
-    <path strokeLinecap="round" strokeLinejoin="round" d="M12 4.5v15m7.5-7.5h-15" />
-  </svg>)
-}
+import React from 'react';
```

Comment:
There are a lot of icons in this folder and they are used at various places and each having a different syntax, props etc. It would be nice if we could either; 1) have a uniform interface manipulatable with tailwindcss classes from the outside. Or 2) separate them into dedicated folders alternatively *name* them to indicate where they are used alternatively *type* them with a type indicating where they are used